src/satellite/test.js

```diff
@@ -988,10 +988,14 @@ describe('Elastic()', () => {
   });
 
   describe('Tests for mock Elastic()', () => {
-    const client = Elastic();
-    const { mock } = client;
+    let client;
+    let mock;
 
-    beforeEach(() => mock.clearAll());
+    beforeEach(() => {
+      client = Elastic();
+      mock = client.mock;
+      mock.clearAll();
```

Comment:
I'm not sure why we need to make this change for tests here. 
If doing so this way, we won't even need `mock.clearAll()` since a new client and mock is created each time
The original tests were to show what would've happened if all mocks were sharing the same client.

Reply:
Thanks for the explanation